uiowa/uiowa
# Facilities - Adjust scope, permissions for alerts, dispatch tab

Resolves https://github.com/uiowa/uiowa/issues/7557

# To Test

```
ddev blt ds --site=facilities.uiowa.edu
```

- As a collaborator/publisher/webmaster visit an alert and see the notifications tab, click it and see the dispatch route.
- As a collaborator/publisher/webmaster visit a page, article or some other piece of content and you won't see the notifications tab.
- As an editor, visit an alert and see that the notifcations tab is not visible to you. Add /dispatch to the node/{nid} of the alert and see that the route is access denied (e.g. https://facilities.uiowa.ddev.site/node/5941/dispatch).
- As a collaborator I can use the reason for closure text area, and upload files and images to the traffic control plan.
